#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Changing d_fmt from "%Y-%m-%d" to "%d.%m.%Y"
# DP: Related bugs: #27397
# DP: Dpatch author: 
# DP: Patch author: Alastair McKinstry
# DP: Upstream status: 
# DP: Status Details: 
# DP: Date: 2002-02-11

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0


# append the patch here and adjust the -p? flag in the patch calls.
Index: glibc-2.6/localedata/locales/de_CH
===================================================================
--- glibc-2.6.orig/localedata/locales/de_CH
+++ glibc-2.6/localedata/locales/de_CH
@@ -114,7 +114,7 @@
         "<U004E><U006F><U0076><U0065><U006D><U0062><U0065><U0072>";/
         "<U0044><U0065><U007A><U0065><U006D><U0062><U0065><U0072>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
-d_fmt   "<U0025><U0059><U002D><U0025><U006D><U002D><U0025><U0064>"
+d_fmt   "<U0025><U0064><U002E><U0025><U006D><U002E><U0025><U0059>"
 t_fmt   "<U0025><U0054>"
 am_pm   "";""
 t_fmt_ampm ""
